app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -414,7 +414,7 @@ class ChatActivity : AppCompatActivity(), IChatView {
     }
 
     override fun databaseUpdated() {
-        rv_chat_feed.smoothScrollToPosition(recyclerAdapter.itemCount)
+        // rv_chat_feed.smoothScrollToPosition(recyclerAdapter.itemCount)
```

Comment:
Why?

Reply:
This line was overwritten. The task, performed by this line is already been done when notifyDatabaseUpdated is called. Repeating the line was leading to the crash of the app, sometimes. So, I commented it out.

Reply:
Remove it then. Commenting out was done in 1980s when there was no version control

Reply:
Updated it.